lib/airbrake.js

```diff
@@ -316,12 +317,12 @@ Airbrake.prototype.addRequestVars = function(request, type, vars) {
 
 Airbrake.prototype.cgiDataVars = function(err) {
   var cgiData = {};
-  Object.keys(process.env).forEach(function(key) {
+  var self = this;
```

Comment:
`forEach` on line 330 should support second argument - `thisArg`, so you don't have to store `this`.